packages/ui/src/Data/Virtualizations/VirtListSkeleton.tsx

```diff
@@ -1,3 +1,5 @@
+// tslint:disable react-unused-props-and-state
```

Comment:
Can we not put this in the global settings?

Reply:
You mean [this line](
https://github.com/syndesisio/syndesis-react-poc/pull/77/files#diff-ace19bd0c04529e685320269e3c05de9R18)? That would remove the rule altogether, which is kinda meh... I found a few problematic areas thanks to it

Reply:
I figured we could add it to the tslint config file, but nevermind. 👍